lib/random.php

```diff
@@ -206,6 +208,7 @@ class_exists('COM')
          */
         function random_bytes($length)
         {
+            unset($length);
```

Comment:
Maybe put an inline comment about it then?
Also, I'm a huge anti-fan of writing code that only satisfies external tools, but just `$length;` may also do the job. :)